src/spatial/detail/ArborX_IndexableGetter.hpp

```diff
@@ -27,29 +27,30 @@ struct DefaultIndexableGetter
   KOKKOS_DEFAULTED_FUNCTION
   DefaultIndexableGetter() = default;
 
-  template <typename Geometry, typename Enable = std::enable_if_t<
-                                   GeometryTraits::is_valid_geometry<Geometry>>>
+  template <typename Geometry>
   KOKKOS_FUNCTION auto const &operator()(Geometry const &geometry) const
   {
     return geometry;
   }
 
-  template <typename Geometry, typename Enable = std::enable_if_t<
-                                   GeometryTraits::is_valid_geometry<Geometry>>>
+  template <typename Geometry,
+            typename Enable = std::enable_if_t<
+                !Details::is_pair_value_index_v<std::decay_t<Geometry>>>>
```

Comment:
This is needed so that universal reference does not grab `PairValueIndex` specializations.